Documentation/Development.md

```diff
@@ -103,41 +103,41 @@ $ .build/x86_64-apple-macosx/debug/spm build
 $ .build/x86_64-apple-macosx/debug/spm test --filter BasicTests.GraphAlgorithmsTests/testCycleDetection
 ```
 
-Note: If you make any changes to `PackageDescription` runtime-related targets,
+Note: If you make any changes to the `PackageDescription` runtime-related targets,
 you **will** need to rebuild using the bootstrap script.
 
 ## Developing using Xcode
 
-Run the following command to generate a Xcode project.
+Run the following commands to generate and open an Xcode project.
 
 ```sh
 $ Utilities/bootstrap --generate-xcodeproj
 generated: ./SwiftPM.xcodeproj
 $ open SwiftPM.xcodeproj
 ```
 
-Note: If you make any changes to `PackageDescription` or `PackageDescription4`
-target, you will need to regenerate the Xcode project using the above command.
+Note: If you make any changes to the `PackageDescription` or `PackageDescription4`
+targets, you will need to regenerate the Xcode project using the above command.
 
 ## Continuous Integration
 
 SwiftPM uses [swift-ci](https://ci.swift.org) infrastructure for its continuous integration testing. The
-bots can be triggered on pull-requests if you have commit access, otherwise ask
+bots can be triggered on pull-requests if you have commit access. Otherwise, ask
 one of the code owners to trigger them for you. The following commands are supported:
 
 ### @swift-ci please smoke test
```

Comment:
This is not related to Hayden's changes, but I find the choice to use a h3 heading for the swift-ci commands a little odd.

Reply:
I changed it so that the swift-ci commands are now displayed as code rather than a heading.